book/text.md

```diff
@@ -559,7 +559,6 @@ self.cursor_x = HSTEP
 self.cursor_y = VSTEP
 self.weight = "normal"
 self.style = "roman"
-self.size = 16
```

Comment:
The `size` field will be introduced later at https://github.com/browserengineering/book/blob/main/book/text.md?plain=1#L633 and is not necessary for the current context.